common.mk

```diff
@@ -103,8 +104,12 @@ QEMU_VIRTFS_MOUNTPOINT	?= /mnt/host
 # Mandatory for autotools (for specifying --host)
 ################################################################################
 ifeq ($(COMPILE_NS_USER),64)
+ifeq ($(UNAME_M),x86_64)
 MULTIARCH			:= aarch64-linux-gnu
 else
+MULTIARCH			:= aarch64-linux
+endif
+else
 MULTIARCH			:= arm-linux-gnueabihf
```

Comment:
I have a doubt about this, given that 
`AARCH32_CROSS_COMPILE := $(AARCH32_PATH)/bin/arm-linux-` below when `!x86_64`. It looks like `MULTIARCH` should be `arm-linux` in this case. Unfortunately I can't test at the moment.

Reply:
I'm testing. It seems that something needs to be changed at least, stay tuned.